packages/cache/package.json

```diff
@@ -47,20 +47,21 @@
     "@types/lodash": "^4.14.108",
     "@types/mocha": "^9.1.1",
     "@types/node": "^18.0.0",
+    "@types/sinon": "^17.0.3",
     "@vitejs/plugin-legacy": "^1.3.2",
-    "chai": "^4.3.6",
     "dexie": "^3.2.2",
     "eventemitter3": "^4.0.7",
     "fake-indexeddb": "^3.1.8",
-    "mocha": "^10.0.0",
     "prettier": "^3.2.5",
     "rimraf": "^3.0.2",
     "semantic-release": "^17.4.2",
+    "sinon": "^18.0.0",
```

Comment:
do we need both sinon and vitest. Vitest provides vi.mock and vi.fn that is used elsewhere in the code base.

Reply:
Actually, I think I am not really using it at all. Will remove.